github/login.go

```diff
@@ -83,3 +92,19 @@ func validateResponse(user *github.User, resp *github.Response, err error) error
 	}
 	return nil
 }
+
+func githubClientFromAuthURL(authURL string, httpClient *http.Client) (*github.Client, error) {
+	if strings.HasPrefix(authURL, "https://github.com/") {
+		return github.NewClient(httpClient), nil
+	} else {
+		baseURL, err := url.Parse(authURL)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse AuthURL %s", authURL)
```

Comment:
Let's use the github prefix and mention the Endpoint type since its what the user will need to correct.

`github: error parsing Endoint.AuthURL: %s`